Jenkinsfile

```diff
@@ -137,20 +137,20 @@ pipeline {
         stage('Release: [ master ]') {
             when {
                 branch 'master'
+                not {
+                    changelog '.*\\[skip ci\\]$'
+                }
+            }
+            environment {
+                GIT_AUTHOR_EMAIL = 'molgenis+ci@gmail.com'
+                GIT_AUTHOR_NAME = 'molgenis-jenkins'
+                GIT_COMMITTER_EMAIL = 'molgenis+ci@gmail.com'
+                GIT_COMMITTER_NAME = 'molgenis-jenkins'
             }
             steps {
-                milestone 1
-                timeout(time: 30, unit: 'MINUTES') {
-                    script {
-                        env.RELEASE_SCOPE = input(
-                                message: 'Do you want to release?',
-                                ok: 'Release'
-                        )
-                    }
-                }
+                milestone 2
                 container('node') {
-                    sh "yarn lerna publish"
-                    hubotSend(message: "${env.REPOSITORY} has been successfully deployed.", status:'SUCCESS')
+                    sh "npx semantic-release -e semantic-release-monorepo"
```

Comment:
We already have semantic release in lerna I think, because we set conventional-commits to true.